bots/naughty/debian-stable/10718-qemu-crash-in-tcg-mode

```diff
@@ -0,0 +1 @@
+Process * (qemu-system-x86) of user * dumped core.
```

Comment:
This is a little too generic IMHO. Can you please include the test case (from the Python backtrace), so that we don't ignore *all* QEMU crashes on Debian stable?

Reply:
Reposted.